syntax = "proto3";

package edu.uci.ics.amber.engine.architecture.worker;
import "edu/uci/ics/amber/engine/architecture/sendsemantics/datatransferpolicy2.proto";
message Pause{}
message Resume{}
message StartWorker{}
message AddOutputPolicy{
  edu.uci.ics.amber.engine.architecture.sendsemantics.DataSendingPolicy policy = 1;
}

message ControlCommand {
  oneof sealed_value {
    Pause pause = 1;
    Resume resume = 2;
    AddOutputPolicy addOutputPolicy = 3;
  }

}

